core/storage/base_model/gae_models.py

```
@@ -2214,3 +2220,79 @@ def _generate_new_id(cls) -> str:
+
+    @classmethod
+    def fetch_page(
+        cls,
+        page_size: int,
+        cursor: Optional[str] = None,
+        author_id: Optional[str] = None,
+        status_filter: Optional[str] = None,
+        exploration_id: Optional[str] = None,
+        date_from: Optional[datetime.datetime] = None,
+        date_to: Optional[datetime.datetime] = None,
+        **kwargs: Any,
```

Ruff (0.15.20)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `**kwargs`

(ANN401)